.travis.yml

```diff
@@ -82,7 +82,7 @@ matrix:
     before_script:
       - du -hs $QTDIR
       - qmake -query
-      - ./installed-files-checker.py --os=macos --install_prefix=${QTDIR} --release ${INSTALL_FILE_CHECKER_ARGS}
+      - ./installed-files-checker.py --os=macos --install_prefix=${QTDIR} --release ${INSTALL_FILE_CHECKER_ARGS} | wc -l
```

Comment:
No, here we need to see file names. ` | tee | wc -l` can do both, though

Reply:
./installed-files-checker.py --os=macos --install_prefix=${QTDIR} --release ${INSTALL_FILE_CHECKER_ARGS} --verbose

Reply:
oh ya sorry will fix it